pkg/apis/forklift/v1beta1/plan/migration.go

```diff
@@ -105,6 +105,7 @@ func (r *Step) ReflectTasks() {
 	tasksCompleted := 0
 	completed := int64(0)
 	if len(r.Tasks) == 0 {
+		r.MarkCompleted()
```

Comment:
can it be that we'll have a step before the tasks are created?
if seems suspicious that we have this if-statement here - it could have been dropped altogether if the intention was to mark the step as completed, no?

Reply:
yes it does. i'll check about having step before tasks. but in my opinion the idea was to not allow a case there are no tasks (no disks).

Reply:
The creation of tasks happens only when entering:
`AllocateDisks, CopyDisks, CopyDisksVirtV2V and Finalize`. In terms of when we enter `ReflectTasks()`, it's when updating the conversion/copy progress. therefore i don't see much risk with this change.

Reply:
I see - so tasks like initialization and hooks won't get it, ok
so how about dropping this if-statement then?

Reply:
done